backend/infrahub/graphql/mutations/branch.py

```diff
@@ -244,23 +244,22 @@ async def mutate(cls, root: dict, info: GraphQLResolveInfo, data: BranchNameInpu
 
             ok = True
 
-            if context.service:
```

Comment:
just removed this unnecessary conditional, it is already handled at  the beginning of the method